docker-compose.yml

```diff
@@ -618,6 +613,7 @@ services:
       - OTEL_RESOURCE_ATTRIBUTES
       - OTEL_SERVICE_NAME=kafka
       - KAFKA_HEAP_OPTS=-Xmx400m -Xms400m
+      - OTEL_INFERRED_SPANS_ENABLED=false
```

Comment:
I tried just changing this in the Dockerfile for the image, but even after deleting the image it was still there.